### ISSUE TITLE
Atom now bundles autocomplete-plus but its not picked up by python-jedi

### ISSUE THREAD

**Issue by ergo** [tinloaf/autocomplete-plus-python-jedi#22]
What description says, latest version of atom comes with bundled autocomplete-plus but upon run python-jedi complains about it not being present despite atom reporting it as working.

```
Autocomplete-Plus-Python-Jedi requires the Autocomplete-Plus package to work, which is not installed. Please install Autocomplete-Plus.
```

**Comment by danqing** [comment]
+1.

I'm not super familiar with Atom code, but I think it's just a one line change of https://github.com/tinloaf/autocomplete-plus-python-jedi/blob/04cf9ea5fac2e9b7236cdb6a09e586f2b6bc402e/lib/acp-python-jedi.coffee#L11

**Comment by v3ss0n** [comment]
made a pull request for this #23

**Comment by tinloaf** [comment]
Jup, thanks for the hint. I had not upgraded my Atom yet and did not see that yet... Fixed in 0.3.4.
